app/pages/summary/complete/ui-tests/withSavedData.ui.test.js

```diff
@@ -343,10 +343,10 @@ test('should render the recurring cost item details in the table', async (t) =>
     .expect(await extractInnerText(recurringCostRow1.find('td').nth(1))).eql('C000001-01-A10001-2')
     .expect(await extractInnerText(recurringCostRow1.find('td').nth(2))).eql('Some catalogue name')
     .expect(await extractInnerText(recurringCostRow1.find('td').nth(3))).eql('SI1-A10001')
-    .expect(await extractInnerText(recurringCostRow1.find('td').nth(4))).eql('500.26 per license')
+    .expect(await extractInnerText(recurringCostRow1.find('td').nth(4))).eql('500.261 per license')
     .expect(await extractInnerText(recurringCostRow1.find('td').nth(5))).eql('12')
     .expect(await extractInnerText(recurringCostRow1.find('td').nth(6))).eql('6 August 2020')
-    .expect(await extractInnerText(recurringCostRow1.find('td').nth(7))).eql('6,003.13')
+    .expect(await extractInnerText(recurringCostRow1.find('td').nth(7))).eql('6,003.132')
```

Comment:
I think it might be worth including a price with 4 d.p. in one or more of the UI tests. (Either this one and/or the UI tests for an incomplete order.)

Reply:
id include one with 5/6 decimal places too